runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java

```diff
@@ -186,35 +226,32 @@ public void makeOverlayHint(PanelComponent panelComponent, ClueScrollPlugin plug
 			panelComponent.getLines().add(new PanelComponent.Line(getSecondEmote().getName(), TITLED_CONTENT_COLOR));
 		}
 
-		if (getItemIds().length != 0)
+		if (getItemRequirements() == null)
+		{
+			panelComponent.setWidth(130);
+			panelComponent.getLines().add(new PanelComponent.Line("Items:", "None"));
+		}
+		else
 		{
 			panelComponent.setWidth(160);
 			panelComponent.getLines().add(new PanelComponent.Line("Equip:"));
 
-			if (plugin.getEquippedItems() != null)
+			Item[] items = plugin.getEquippedItems();
```

Comment:
Probably MoreObjects.firstNonNull would be cleaner, and storing the new Items[0] as constant

Reply:
What would the gain be from using firstNonNull?

Reply:
Nothing, it's just cleaner a bit